package/MDAnalysis/converters/RDKit.py

```diff
@@ -120,31 +107,11 @@
         "segindices": "SegmentNumber",
         "tempfactors": "TempFactor",
     }
-    PERIODIC_TABLE = Chem.GetPeriodicTable()
-
-
-# charges that should be assigned to monatomic cations
-# structure --> atomic number : formal charge
-# anion charges are directly handled by the code using the typical valence
-# of the atom
-MONATOMIC_CATION_CHARGES = {
```

Comment:
note to self: this and `STANDARDIZATION_REACTIONS` should be kept available for backwards compatibility (can use the same pattern as below with the `DEFAULT_INFERER`